kubernetes_platform/python/generate_proto.py

```diff
@@ -69,7 +69,6 @@ def generate_proto(source: str) -> None:
             f'-I={PROTO_DIR}',
             f'--experimental_allow_proto3_optional',
             f'--python_out={PKG_DIR}',
-            pipeline_api_source,
```

Comment:
Does this still need to be defined on line 27?

Reply:
ah yeah this one is included in the "update to protoc 3.20" commit